ee/tabby-db/migrations/08-repositories-table/up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE repositories (
```

Comment:
Again, please follow example in issue to define the table:
https://github.com/TabbyML/tabby/issues/1143